### PR TITLE
IGNITE-17113 Added info on distributed computing

### PR DESCRIPTION
**Pull request by IgGusev**

### CODE REVIEW

File: docs/_docs/compute/compute.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+private static class NodeNameJob implements ComputeJob<String> {
+    @Override
+    public String execute(JobExecutionContext context, Object... args) {
+        return context.ignite().name() + "_" + args[0];
+    }
+}
+----
+
+NOTE: Unlike Ignite 2, jobs are currently not serialized.
```
[the remark this answers is not in the document]
Comment (ptupitsyn) [reply docs/_docs/compute/compute.adoc:47]: There are no plans to serialize jobs for now, so let's drop "currently".

File: docs/_docs/compute/compute.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+== .NET: Thin
```
Comment (ptupitsyn) [inline docs/_docs/compute/compute.adoc:49]:
```suggestion
== .NET
```

File: docs/_docs/compute/compute.adoc [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+[source, csharp]
+----
+IIgniteClient client = Client;
+ICompute compute = client.Compute;
+IList<IClusterNode> nodes = await Client.GetClusterNodesAsync();
+string res = await compute.ExecuteAsync<string>(nodes, jobClassName: "org.foo.bar.NodeNameJob", "Hello!");
+----
+
+NOTE: Unlike Ignite 2, jobs are currently not serialized.
```
Comment (ptupitsyn) [inline docs/_docs/compute/compute.adoc:63]:
```suggestion

```

Comment (ptupitsyn) [reply docs/_docs/compute/compute.adoc:63]: The API only accepts the Java job class name, so there is no possibility of job serialization. I'm not sure if this remark makes sense here.

**Review by ptupitsyn — CHANGES_REQUESTED** [review]

File: docs/_docs/compute/compute.adoc [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+executeSql("INSERT INTO test(k, v) VALUES (1, 101)");
+----
+
+[tabs]
+--
+tab:Tuple[]
+[source, java]
+----
+String actualNodeName = ignite.compute()
+        .executeColocated("PUBLIC.test", Tuple.create(Map.of("k", 1)), GetNodeNameJob.class)
```
Comment (rpuch) [inline docs/_docs/compute/compute.adoc:65]: Should it be replaced with a call to `NodeNameJob`? If not, then `GetNodeNameJob` code is missing.

File: docs/_docs/compute/compute.adoc [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        .get(1, TimeUnit.SECONDS);
+
+System.out.println(actualNodeName);
+----
+
+tab:Mapper[]
+[source, java]
+----
+String actualNodeName = ignite.compute()
+        .executeColocated("PUBLIC.test", 1, Mapper.of(Integer.class), GetNodeNameJob.class)
```
Comment (rpuch) [inline docs/_docs/compute/compute.adoc:75]: Same thing about the job

**Review by ptupitsyn — CHANGES_REQUESTED** [review]

**Review by ptupitsyn — APPROVED** [review]
Looks good to me :+1:

**Review by rpuch — APPROVED** [review]

**Pull request closed without merging**